src/ontology/mondo-edit.obo

```diff
@@ -513981,6 +513981,57 @@ relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7100" xsd:anyURI
 
+[Term]
+id: MONDO:0100539
+name: hemiplegic migraine-developmental and epileptic encephalopathy spectrum
+def: "A spectrum in which individuals may present with phenotypes ranging from hemiplegic migraines without epilepsy to developmental and epileptic encephalopathy with or without episodic hemiplegia or other forms of paresis. Symptoms and severity may vary within families." [https://clinicalgenome.org/affiliation/40005/]
+is_a: MONDO:0018925 {source="https://clinicalgenome.org/affiliation/40005/"} ! familial or sporadic hemiplegic migraine
+is_a: MONDO:0100062 {source="https://clinicalgenome.org/affiliation/40005/"} ! developmental and epileptic encephalopathy
+property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7195" xsd:anyURI
+
+[Term]
+id: MONDO:0100540
+name: GATA6-related congenital heart disease with or without pancreatic agenesis or neonatal diabetes
+def: "A congenital heart disease that is present at birth. Representative examples include atrial septal defect 9, conotruncal heart malformations, tetralogy of Fallot, ventricular septal defect, atrioventricular septal defect, bicuspid aortic valve, transposition of the great arteries, persistent truncus arteriosus, congenital heart disease with pancreatic agenesis, and congenital heart disease with neonatal diabetes." [https://clinicalgenome.org/affiliation/40130/, PMID:19666519, PMID:23020118, PMID:25937001, PMID:28049534, PMID:29653232, PMID:31301121, PMID:35595280]
+synonym: "GATA6 related congenital heart disease with or without pancreatic agenesis or neonatal diabetes" EXACT [https://clinicalgenome.org/affiliation/40130/]
+is_a: MONDO:0005453 {source="https://clinicalgenome.org/affiliation/40130/"} ! congenital heart disease
+relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/4174 {source="https://clinicalgenome.org/affiliation/40130/"} ! GATA6
+property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7218" xsd:anyURI
+
+[Term]
+id: MONDO:0100541
+name: GATA5-related congenital heart defects
+def: "A congenital heart disease that is present at birth. Representative examples include tetralogy of fallot, bicuspid aortic valve, atrial septal defect, double outlet right ventricle, ventricular septal defect, and coarctation of the aorta, and atrioventricular canal." [https://clinicalgenome.org/affiliation/40130/, PMID:22641149, PMID:22961344, PMID:23031282, PMID:23289003, PMID:24638895]
+synonym: "GATA5 related congenital heart defects" EXACT [https://clinicalgenome.org/affiliation/40130/]
+is_a: MONDO:0005453 {source="https://clinicalgenome.org/affiliation/40130/"} ! congenital heart disease
+property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7242" xsd:anyURI
+
+[Term]
+id: MONDO:0100542
+name: clonal hematopoiesis
```

Comment:
See comment in issue: https://github.com/monarch-initiative/mondo/issues/7137#issuecomment-1933129034